src/button/high-visibility.scss

```diff
@@ -10,22 +11,26 @@
     color: t(iui-color-foreground-accessory);
   }
 
-  > .iui-icon {
+  &[disabled] {
```

Comment:
disabled selector should come *after* all the hover/focus/active states (a later rule takes preference if they have the same specificity).

Should be fixed in all files